js/directDebitInformation.js

```diff
@@ -0,0 +1,51 @@
+CRM.$(function ($) {
```

Comment:
but this js file do nothing currently, right ? because I don't see you adding it to any page in the PR code, or am I miss something ?

Reply:
This JS file was added at hook manualdirectdebit_civicrm_pageRun(&$page) in manualdirectdebit.php file.

Reply:
oh okay